lib/matplotlib/lines.py

```diff
@@ -790,6 +790,9 @@ def draw(self, renderer):
                     cap = self._dashcapstyle
                     join = self._dashjoinstyle
                 else:
+                    print(tpath.vertices.size)
+                    tpath = affine.inverted().transform_path(affine.transform_path(tpath).cleaned(simplify=True))
+                    print(tpath.vertices.size)
```

Comment:
@anntzer are you saying if I edit `_classic_test.mplstyle` to turn path simplification off, that all the figures created during testing will be made with simplification turned off?